oracle/assets/configuration/spec.yaml

```
@@ -38,5 +50,24 @@ files:
       value:
         type: string
         example: <DRIVER_PATH>.jar
+    - name: jdbc_truststore_path
+      description: |
+        The JDBC truststore file path.
+
+        Note: This option is required if connecting to TCPS via JDBC.
```

```suggestion
        Note: This option is required if connecting with TCPS via JDBC.
```

this is a typo on my previous suggestion